here/traffic/here_api.py

```diff
@@ -51,7 +51,13 @@ def get_access_token(key_id, key_secret, token_url):
         r.raise_for_status()
     except KeyError as k_err:
         LOGGER.error('Key Error in getting access token, response was') 
-        raise HereAPIException(r.text)
+        LOGGER.error(k_err)
+        try: 
+            err_msg = r.json()['message'] 
+        except JSONDecodeError:
+            err_msg = r.text
+        finally:
+            LOGGER.error(err_msg)
```

Comment:
stil need `raise HereAPIException(r.text)` otherwise the script won't actually fail properly here.